lib/datasource/rubygems/get-rubygems-org.ts

```diff
@@ -15,38 +15,69 @@ export function resetCache(): void {
   contentLength = 0;
 }
 
-/* https://bugs.chromium.org/p/v8/issues/detail?id=2869 */
-const copystr = (x: string): string => (' ' + x).slice(1);
+export class RubyGemsOrgDatasource extends Datasource {
+  constructor(override readonly id: string) {
```

Comment:
I still think we should share the `http` instance between those internal datasources. 🤔